mvcc/backend/backend.go

```diff
@@ -25,7 +25,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/boltdb/bolt"
+	bolt "github.com/coreos/bbolt"
```

Comment:
I am curious on why naming the import `github.com/coreos/bbolt` to `bolt`?

Reply:
nvm, I see what's going on.

Reply:
Just to minimize the code change-- this way I only need to change the import, no any of the package references in the rest of the go file.

Reply:
For master this wouldn't make as much sense since that code lives on indefinitely, but for a backport to a branch that has a finite livespan it simplifies things slightly.